### PR TITLE
chore: Enable renovate (for js/ts) and dependabot (for github actions)

### PR DESCRIPTION
**Pull request by tumido**
1. Enables https://github.com/renovatebot/renovate to keep JS deps up to date
2. Enabled [dependabot](https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/about-dependabot-version-updates) for Github Actions

### CODE REVIEW

File: .github/renovate.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "extends": [
+        "config:js-app"
```
Comment (harshad16) [inline .github/renovate.json:3]: Does this mean we are stating it a javascript app or `js-app` means something else?
couldn't find details here: https://docs.renovatebot.com/configuration-options/#extends

**Review by harshad16 — COMMENTED** [review]
/lgtm
just one question

File: .github/renovate.json (hunk 1 above)
Comment (tumido) [reply .github/renovate.json:3]: Their documentation is not very well organized. :slightly_smiling_face: You need to look here: https://docs.renovatebot.com/presets-config/#configjs-app

It's a preset that should give us some reasonable base behavior for JS apps (compared to JS libraries where dependency pinning is undesirable etc.).

Comment (harshad16) [reply .github/renovate.json:3]: ack, thanks for the knowledge share.

**Review by harshad16 — APPROVED** [review]
lgtm
thanks

**Pull request merged**